### ISSUE TITLE
java.lang.NoSuchMethodError when trying to post to phabricator

### ISSUE THREAD

**Issue by Melvillian** [jenkinsci/cobertura-plugin#73]
We run our test suite using jenkins, and recently we started seeing this error in our Post to Phabricator post-build step

```
ERROR: Step ‘Post to Phabricator’ aborted due to exception: 
java.lang.NoSuchMethodError: hudson.plugins.cobertura.targets.CoverageResult.setOwner(Lhudson/model/AbstractBuild;)V
	at com.uber.jenkins.phabricator.coverage.CoberturaCoverageProvider.computeCoverage(CoberturaCoverageProvider.java:131)
	at com.uber.jenkins.phabricator.coverage.CoberturaCoverageProvider.hasCoverage(CoberturaCoverageProvider.java:61)
	at com.uber.jenkins.phabricator.PhabricatorNotifier.getCoverageProvider(PhabricatorNotifier.java:252)
	at com.uber.jenkins.phabricator.PhabricatorNotifier.perform(PhabricatorNotifier.java:98)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:730)
	at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:676)
	at hudson.model.Build$BuildExecution.post2(Build.java:186)
	at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:621)
	at hudson.model.Run.execute(Run.java:1760)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:415)
Finished: FAILURE
```

This error still persists even after unchecking the "Enable Uberalls" option in the builds "Configure" tab. We are using plugin version 1.10, and for some reason we can't uninstall it (the option is greyed out)

What do we need to do to rid us of this Runtime Error?

**Comment by jeffpearce** [comment]
@Melvillian @sujeet - could someone try building #77 and let me know if it fixes everything? It fixes the exception reported, but I don't have the end-to-end configured, so I don't know if there are other issues lurking.

**Comment by jeffpearce** [comment]
This is fixed in release 1.12. If you still see problems, please either tag me here or open a new issue,
